DFC.ServiceTaxonomy.CustomFields/Manifest.cs

```diff
@@ -0,0 +1,10 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "DFC.ServiceTaxonomy.CustomFields",
+    Author = "The Orchard Team",
+    Website = "https://orchardproject.net",
+    Version = "0.0.1",
+    Description = "DFC.ServiceTaxonomy.CustomFields",
+    Category = "Content Management"
```

Comment:
Could you add any dependencies into the manifest (OrchardCore.ContentTypes?).
Also, could you supply suitable values for the other properties